src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,49 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/** Deadline is a variation of Tasks that carries a task description and its deadline
+ * @author CShuwen
+ * @version 1.0
+ * @since 0.0
+ */
+public class Deadline extends Task{
+    private int isDone;
+    private final LocalDate by;
+
+    public Deadline(String description, LocalDate by, Integer isDone){
+        super(description, isDone);
+        this.by = by;
+        this.isDone = isDone;
+    }
+
+    @Override
+    public void unMark(){
```

Comment:
nitpicking but please add a space between reserved words and the Egyptian braces. I noticed the same minor issue in several other places also. 😓